package.json

```diff
@@ -12,7 +12,7 @@
     "postinstall": "lerna bootstrap",
     "build": "gulp",
     "build-all": "gulp all",
-    "release": "lerna bootstrap; gulp; lerna publish --skip-git; gulp version;",
+    "release": "lerna bootstrap; lerna publish --yes --cd-version prerelease --exact --skip-npm --skip-git; gulp version; gulp; git checkout lerna.json; lerna publish --cd-version prerelease  --exact --skip-git",
```

Comment:
This one I don’t understand.if you call publish twice it will increase by two versions. Seems the only thing you had todo is add —exact
We can’t hardcode prelrelease, as in the future I will also release other versions.
For what is the git checkout lerna.json?
This needs to be simplified, currently it will probably create more issues than it solves.

Reply:
OK, now I'm confused as well.  I added the complexity as a way to work around an issue I was seeing where the built web3.min.js contained out-of-date version numbers for some packages.

I've reverted to the simpler command-line with just the addition of `--exact`, and I can't reproduce the issue I had earlier, so I'll commit the simpler line.